crml/cennzx-spot/rpc/runtime-api/src/lib.rs

```diff
@@ -1,18 +1,17 @@
-// Copyright 2020 Parity Technologies (UK) Ltd. and Centrality Investments Ltd.
-// This file is part of Substrate.
-
-// Substrate is free software: you can redistribute it and/or modify
-// it under the terms of the GNU General Public License as published by
-// the Free Software Foundation, either version 3 of the License, or
-// (at your option) any later version.
-
-// Substrate is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-// GNU General Public License for more details.
-
-// You should have received a copy of the GNU General Public License
-// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+/* Copyright 2019-2020 Centrality Investments Limited
```

Comment:
The existing license header was ok for this file, since it's modified from Substrate